packages/core/src/getFilesList.ts

```diff
@@ -179,6 +179,29 @@ export type GetFilesListArgs = {
   extensionTester?: RegExp
 }
 
+const findRepoRoot = async ({
+  searchRoot,
+  fsRoot,
+}: {
+  searchRoot: string
+  fsRoot: string
+}) => {
+  let currentDir = path.resolve(searchRoot)
```

Comment:
Not sure if the code should be defensive - if the user passes a path that doesn't exist this will throw an error